libs/sdk-ui-pivot/src/impl/data/rowFactory.ts

```diff
@@ -324,27 +331,78 @@ export function createAgGridPage(
 
     const minimalRowData: DataValue[][] = getMinimalRowData(dv);
 
-    const columnTotalsData = dv.rawData().columnTotals();
-
-    const subtotalStyles = getSubtotalStyles(dimensions?.[0]);
-    const rowData = minimalRowData.map((dataRow: DataValue[], dataRowIndex: number) => {
-        const mergedDataRowWithColumnTotals = dv.rawData().hasColumnTotals()
-            ? [...dataRow, ...columnTotalsData![dataRowIndex]]
-            : dataRow;
-        return getRow(
-            tableDescriptor,
-            mergedDataRowWithColumnTotals,
-            dataRowIndex,
-            headerItems[0],
-            subtotalStyles,
-            intl,
-        );
-    });
+    if (tableDescriptor.headers.mixedHeadersCols.length > 0 && tableDescriptor.isTransposed()) {
+        const rowData: IGridRow[] = [];
 
-    const rowTotals = getRowTotals(tableDescriptor, dv, intl)!;
+        // rows with attribute values
+        headerItems[1].forEach((attributes, rowIndex) => {
+            const headerColumn = tableDescriptor.headers.mixedHeadersCols[0];
+            const attributeName = dv.data().slices().toArray()[0].descriptor.descriptors[rowIndex]
+                .attributeHeader.formOf.name;
 
-    return {
-        rowData,
-        rowTotals,
-    };
+            const row: IGridRow = {
+                [headerColumn.id]: attributeName,
+                headerItemMap: {},
+            };
+
+            tableDescriptor.headers.mixedValuesCols.forEach((column, columnIndex) => {
+                const header = attributes[columnIndex];
+                if (isResultAttributeHeader(header)) {
+                    // TODO what about formattedName?
```

Comment:
what should we do with this TODO?

Reply:
formatted name already used with precedents inside getAttributeHeaderItemName. Removing TODO